takoyaki/src/main/java/frc/robot/Constants.java

```diff
@@ -29,7 +29,7 @@
   }
   
   public static final class kHopper {
-    public static final int MOTOR_ID = -1;
+    public static final int MOTOR_ID = 10;
     public static final boolean INVERTED = false;
     public static final int CURRENT_LIMIT = 30;
     public static final double SPEED = 0.9;
```

Comment:
```suggestion
    public static final double SPEED_PERCENT = 0.9;
```

Consider consistency with `INTAKE_SPEED` vs just `SPEED` here